### PR TITLE
Add SailthruApi class and configure it in script helpers.

### PR DESCRIPTION
**Pull request by doctoryes**
https://openedx.atlassian.net/browse/PLAT-2364

Call the Sailthru API to delete a user by email address.

### CODE REVIEW

File: tubular/sailthru_api.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        Delete a user from Sailthru using their email address.
+        """
+        email = learner.get('original_email', None)
+        if not email:
+            raise TypeError('Expected an email address for user to delete, but received None.')
+
+        try:
+            sailthru_response = self._sailthru_client.api_delete("user", {'id': email})
+        except SailthruClientError as exc:
+            error_msg = "Exception attempting to delete user {} from Sailthru - {}".format(email, text_type(exc))
```
Comment (pwnage101) [inline tubular/sailthru_api.py]: Is there any reason to expect non-ascii from these API error messages?  if so, just a reminder that python 2 scripts in jenkins break unless we respect the limitations of that environment (only encoded utf-8 bytes, or ascii-only strings going to stdout): https://github.com/edx/tubular/blob/master/tubular/google_api.py#L6-L10

**Review by pwnage101 — APPROVED** [review]
LGTM, just one thought.

**Review by bmedx — APPROVED** [review]

File: tubular/sailthru_api.py (hunk 1 above)
Comment (doctoryes) [reply tubular/sailthru_api.py]: Good point - I'll add this to the log messages.

**Comment by pwnage101** [comment]
:+1:

**Pull request merged**